python/tvm/relay/prelude.py

```diff
@@ -383,6 +383,12 @@ def define_tensor_array_scatter(self, indices_shape=None, force_update=False):
             return
 
         tensor_array_scatter_helper_name = self.get_name("tensor_array_scatter_helper")
+
+        if indices_shape:
```

Comment:
There is an ```force_update``` argument in tensor_array_scatter API. It would be better to add a test in test_adt.py and figure out why it is not updated in this case, rather than just modify the name. 

Also tensor_array_split can have similar issue.

Reply:
Please see this line
https://github.com/apache/incubator-tvm/blob/7649075fbb71ecab0a41c6fe4d41a86724e42e7a/python/tvm/relay/prelude.py#L425
force_update works, and scatter funciton in mod is replaced, but the original one for first scatter will be needed too.

Consider the case I added, there are two scatters in this case. If we don't put indices into name, tensor_array_scatter_var will be the same for two scatters, but Function will be different. The first function is overrided by the second one, and the second Function can match the first scatter's type.

Reply:
In this case, we might want to enhance ```get_name``` API to allow pass in extra argument shape info. Note that current tensor array op name is in the form of ```tensor_array_xxx_dtype_shape```. If multiple shapes are added, we need to add extra argument name, such as ```indice``` to make the name clear.